alacritty_terminal/src/term/mod.rs

```diff
@@ -1812,18 +2097,21 @@ impl<T: EventListener> ansi::Handler for Term<T> {
                 for cell in &mut row[col..] {
                     cell.reset(&template);
                 }
+                self.damage.expand_line_damage(self.cursor.point.line, col, self.grid.num_cols());
```

Comment:
are you including your right upper bound or not? in first case it should be `num_cols() - 1`, since cols are zero based, in the second case each damage like `expand_line_damage(line, col, col)`, should be `line, col, col + 1`.

Reply:
Ah, supposed to be `-1`. Hidden by the fact that excess damage is inconsequential. Nice catch. :)